Okta.Core/IOktaHttpClient.cs

```diff
@@ -18,10 +18,10 @@ public enum HttpRequestType
         }
 
         public virtual string ApiToken { get; set; }
-        public virtual Uri BaseUri { get; set; }
-
-        public abstract HttpResponseMessage Execute(HttpRequestType requestType, Uri uri = null, string relativeUri = null, string content = null, int waitMillis = 0, int retryCount = 0, bool bAddAuthorizationHeader = true);
-        public abstract Task<HttpResponseMessage> ExecuteAsync(HttpRequestType requestType, Uri uri = null, string relativeUri = null, string content = null, int waitMillis = 0, bool bAddAuthorizationHeader = true);
+        public virtual Uri BaseUri { get; set; }
+
+        public abstract HttpResponseMessage Execute(HttpRequestType requestType, Uri uri = null, string relativeUri = null, string content = null, int waitMillis = 0, int retryCount = 0, bool bAddAuthorizationHeader = true, bool bAddXForwardedHeader = false);

```

Comment:
@nbarbettini Since we are passing multiple headers, what about passing a dictionary instead?